spec/default-file-icons-spec.coffee

```diff
@@ -42,24 +41,14 @@ describe 'DefaultFileIcons', ->
       filePath = path.join(tempDir, 'foo.bar')
       linkPath = path.join(tempDir, 'link.bar')
       fs.writeFileSync(filePath, '')
-      try
```

Comment:
I put these in because they fail unless you're an Admin - probably want to keep them in as AppVeyor etc. do not run as admin